internal/agents/kimi/store.go

```
@@ -0,0 +1,755 @@
+		Provider: account.ProviderKimi,
+		AuthMode: account.AuthModeOAuth,
+		Label:    label,
+		Token:    credential.AccessToken,
+		Source:   source,
+	}
+}
+
+// writeCredential persists the refreshed credential over the CLI's file,
+// atomically and private, so a crash mid-write cannot truncate the only copy
+// of the refresh token.
+func (s Store) writeCredential(credential CredentialInfo) error {
```

golangci-lint (2.12.2)

[error] func Store.writeCredential is unused

(unused)